src/app/pages/FrontPage/index.jsx

```diff
@@ -95,11 +95,11 @@ const FrontPage = ({ pageData, mostReadEndpointOverride }) => {
         openGraphType="website"
       />
       <LinkedData type="WebPage" seoTitle={seoTitle} />
+      <AdContainer />
       <main role="main">
         <VisuallyHiddenText id="content" tabIndex="-1" as="h1">
           {offScreenText}
         </VisuallyHiddenText>
```

Comment:
The skip to content links to the `H1` so I think we can still keep the Ad inside the `main`:
```suggestion

      <main role="main">
        <AdContainer />
        <VisuallyHiddenText id="content" tabIndex="-1" as="h1">
          {offScreenText}
        </VisuallyHiddenText>
```

Reply:
Semantically the advert isn't main content. This should be semantically correct, as I believe there will be times when this content is accessible e.g. when we provide our own content here... So the leaderboard advert should be in a region landmark e.g. section HTML element with a  ```role="region"``` attribute and this will also need a label (I guess we can aria-labelledby to the 'advertisement' link and use that as the label? Though when there is more than 1 advert on the page that wouldn't work... And I think there will be more than one at some stage? Though would there ever be more than 1 time when we provide our own content?